components/src/core/InfoListItem/InfoListItem.tsx

```diff
@@ -320,12 +313,12 @@ const InfoListItemRender: React.ForwardRefRenderFunction<unknown, InfoListItemPr
             dense={props.dense}
             ripple={ripple}
             iconColor={iconColor}
-            className={cx(combine('root'), userClassName)}
+            className={(generatedClasses.root, userClassName)}
```

Comment:
Here's the reason the DrawerNavItems look weird...this syntax is invalid
```suggestion
            className={cx(generatedClasses.root, userClassName)}
```